editor/editor_export.cpp

```diff
@@ -509,7 +510,7 @@ void EditorExportPlatform::_edit_filter_list(Set<String> &r_list, const String &
 
 	DirAccess *da = DirAccess::open("res://");
 	ERR_FAIL_NULL(da);
-	_edit_files_with_filter(da, filters, r_list, exclude);
+	_edit_files_with_filter(da, filters, r_list, exclude); //some renaming, that's all
```

Comment:
Informal notes like this probably don't belong in the final code - prefer to delete but potentially reword if you think it adds value - but on the line above.

Reply:
I just needed it for myself, I'll delete all the comments on editor_export.cpp before I merge this PR